src/main/java/edu/harvard/iq/dataverse/harvest/server/web/servlet/OAIServlet.java

```diff
@@ -180,6 +180,11 @@ private RepositoryConfiguration createRepositoryConfiguration() {
         String dataverseName = dataverseService.findRootDataverse().getName();
         String repositoryName = StringUtils.isEmpty(dataverseName) || "Root".equals(dataverseName) ? "Test Dataverse OAI Archive" : dataverseName + " Dataverse OAI Archive";
         InternetAddress internetAddress = MailUtil.parseSystemAddress(settingsService.getValueForKey(SettingsServiceBean.Key.SystemEmail));
+        String scheme = settingsService.getValueForKey(SettingsServiceBean.Key.Scheme, "oai");
+        String repositoryIdentifier = settingsService.getValueForKey(SettingsServiceBean.Key.RepositoryIdentifier, "dataverse.org");
+        String delimiter = settingsService.getValueForKey(SettingsServiceBean.Key.Delimiter, ":");
+        String sampleIdentifier = settingsService.getValueForKey(SettingsServiceBean.Key.SampleIdentifier, "doi:10.7910/DVN/1HE30F");
```

Comment:
In http://oval.base-search.net this DOI is giving an error:

 ERROR: Identify response well-formed but invalid: Element '{http://www.openarchives.org/OAI/2.0/oai-identifier}sampleIdentifier': [facet 'pattern'] The value 'doi:10.7910/DVN/1HE30F' is not accepted by the pattern 'oai:[a-zA-Z][a-zA-Z0-9\-]*(\.[a-zA-Z][a-zA-Z0-9\-]*)+:[a-zA-Z0-9\-_\.!~\*'\(\);/\?:@&=\+$,%]+'., line 5

In context:

<img width="814" alt="Screen Shot 2020-07-21 at 12 23 12 PM" src="https://user-images.githubusercontent.com/21006/88080390-00872a80-cb4d-11ea-9301-2e9865ba0fb4.png">